doc/classes/Button.xml

```diff
@@ -78,6 +78,7 @@
 		</member>
 		<member name="icon" type="Texture2D" setter="set_button_icon" getter="get_button_icon">
 			Button's icon, if text is present the icon will be placed before the text.
+			To edit margin and spacing of the icon, use [code]hseparation[/code] theme property of [Button] and [code]content_margin_*[/code] properties of the used [StyleBox]es.
```

Comment:
I guess we still lack support for `[theme_item hseparation]` even after #29868 ?

Reply:
Yeah.

Reply:
I have a suspicion that operators and constructors don't have their own tags either :eyes:  (And they are also missing from doc status).